lib/src/model/extension.dart

```diff
@@ -13,63 +11,64 @@ import 'package:dartdoc/src/quiver.dart' as quiver;
 
 /// Extension methods
 class Extension extends Container implements EnclosedElement {
-  ElementType extendedType;
+  ElementType? extendedType;
 
   Extension(
       ExtensionElement element, Library library, PackageGraph packageGraph)
       : super(element, library, packageGraph) {
-    extendedType = modelBuilder.typeFrom(_extension.extendedType, library);
+    extendedType = modelBuilder.typeFrom(element.extendedType, library);
```

Comment:
`extendedType` might be made non-nullable and final by just moving this statement into a constructor initializer.